recipes-core/images/seapath-guest-common.inc

```diff
@@ -14,6 +14,9 @@ IMAGE_INSTALL_append = " \
     net-snmp-server \
     net-snmp-client \
     net-snmp-dev \
+    ethtool \
+    chrony \
```

Comment:
You can also configure chrony to use the PTP clock.
To do this create the file:
*recipes-support/chrony/chrony_%.bbappend*
```
FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
```

*recipes-support/chrony/files/chrony.conf*
```
refclock PHC /dev/ptp0 poll 2
```